### PR TITLE
Issue #mentor : feat : profile and edit profile added

### PR DESCRIPTION
**Pull request by pradeepkumarcm96**

### CODE REVIEW

File: src/app/pages/edit-profile/edit-profile.page.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+<ion-content [fullscreen]="true">
+  <app-profile-image [profileImageData]='profileImageData'></app-profile-image>
+  <div>
+    <app-dynamic-form [jsonFormData]="formData" #form1> </app-dynamic-form>
+  </div>
+</ion-content>
+
+<ion-footer class="ion-no-border" submitButton>
+  <ion-toolbar>
+    <ion-button slot="end" (click)="onSubmit()">Submit</ion-button>
```
Comment (kiranharidas187) [inline src/app/pages/edit-profile/edit-profile.page.html:12]: translate

File: src/app/pages/tabs/profile/profile.page.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<app-page-header [config]="headerConfig"></app-page-header>
+
+<ion-content>
+<ion-button shape="round" expand="full" (click)="editProfile()">Edit Profile</ion-button>
```
Comment (kiranharidas187) [inline src/app/pages/tabs/profile/profile.page.html:4]: trasnlate

File: src/assets/dummy/profile-form.json [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+        }
+      ]
+    },
+    {
+      "name": "experience",
+      "label": "Your Experience in years",
+      "value": "",
+      "class": "ion-margin",
+      "type": "tel",
+      "position": "floating",
```
Comment (rejneesh1) [inline src/assets/dummy/profile-form.json:67]: type should be number @pradeepkumarcm96

File: src/app/pages/tabs/profile/profile.page.html (hunk 2 above)
Comment (pradeepkumarcm96) [reply src/app/pages/tabs/profile/profile.page.html:4]: this button is only for testing

**Pull request merged**